tests/test_chi_school_community_action_council

```diff
@@ -2,10 +2,10 @@
 import pytest
 
 from tests.utils import file_response
-from city_scrapers.spiders.cps_community_action_council import Cps_community_action_councilSpider
+from city_scrapers.spiders.chi_school_community_action_council import chi_school_community_action_councilSpider
```

Comment:
Small thing, but it needs to be the same renamed class name with a capital C here, `Chi_school_community_action_councilSpider`